.editorconfig

```diff
@@ -0,0 +1,30 @@
+# http://editorconfig.org
+
+[*]
+charset = utf-8
+end_of_line = lf
+trim_trailing_whitespace = true
+insert_final_newline = true
+indent_style = space
+indent_size = 4
+
+[*.{html,json}]
+indent_style = space
+indent_size = 2
+trim_trailing_whitespace = false
+
+[*.{scss,js}]
```

Comment:
Currently we don't have any custom JS code. It just wasn't necessary so far.
Is there a reason to have tabs specifically in JS and CSS/SCSS (besides that tabs allow indentation of any size)? :)

And since start it was decided to use simply CSS and not a preprocessed language. So probably should be changed to `css`.